assets/src/js/media_archive_editor_view.js

```diff
@@ -0,0 +1,65 @@
+import {createRoot} from '@wordpress/element';
+import ArchivePicker from '../js/Components/ArchivePicker/ArchivePicker';
+
+const frame = wp.media.view.MediaFrame.Select;
+
+(function() {
+  wp.media.view.MediaFrame.Select = frame.extend({
+    initialize() {
+      frame.prototype.initialize.apply(this, arguments);
+
+      const State = wp.media.controller.State.extend({
+        insert() {
+          this.frame.close();
+        },
+      });
+
+      this.states.add([
+        new State({
+          id: 'mediaArchive',
+          search: false,
+          title: 'Media Archive',
+        }),
+      ]);
+
+      this.on('content:render:mediaArchive', this.renderMediaArchive, this);
+    },
+
+    browseRouter(routerView) {
+      routerView.set({
+        upload: {
+          text: wp.media.view.l10n.uploadFilesTitle,
+          priority: 20,
+        },
+        mediaArchive1: {
```

Comment:
is it intentional `mediaArchive1`?

Reply:
An error, my plan originally was to create two different pop ups, but does not seem like it is needed at the moment.